udata_front/tests/views/test_organization.py

```diff
@@ -284,7 +276,6 @@ def test_resources_csv(self):
             for _ in range(3)
         ]
         not_org_dataset = DatasetFactory(resources=[ResourceFactory()])
-        hidden_dataset = DatasetFactory()
```

Comment:
We now have a `HiddenDatasetFactory` with private=True if you wanna keep tests on hidden datasets?

Reply:
I'm not sure I'm a big fan of `HiddenDatasetFactory`. I start using it but is it really useful to abstract away the fact that it's just `private=True`?

Reply:
And yes you're right I need to add back the test :-) (didn't understood your comment)

Reply:
Done in 5241907a